### PR TITLE
#117 changed datetime field

### LINKED ISSUE #117: Change data type of datetime in order request?
Does it make sense that the datetime in POST /orders is a array instead of a string as in STAC search? This would be easier to validate in JSON Schema and OpenAPI.

### PR DESCRIPTION
**Pull request by captaincoordinates**
Closes #117

### CODE REVIEW

File: order/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Ordering with loosely defined order values will give the provider more freedom t
 ## Request
 | Field Name | Type                                                                       | Description |
 | ---------- | -------------------------------------------------------------------------- | ----------- |
-| datetime       | string                                                                     | **REQUIRED.** Datetime field is a [ISO8601 Time Interval](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) |
+| datetime       | string                                                                     | **REQUIRED.** Single date+time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots .. for open date ranges. \[Copied from [STAC API spec](https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/item-search#query-parameter-table)\] |
```
Comment (m-mohr) [inline order/README.md:12]:
```suggestion
| datetime       | string                                                                     | **REQUIRED.** Single date and time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots `..` for open date ranges. Only one boundary can use double dots. |
```

Comment (m-mohr) [reply order/README.md:12]: What does providing a single datetime mean?

Comment (captaincoordinates) [reply order/README.md:12]: Good question. I imagine it means the same as providing `..` as the second part. But maybe we need to define a new data type here that mandates both parts. *Or*... as previously suggested - make this an array that requires length of 2. This seems like a stronger argument now

Comment (m-mohr) [reply order/README.md:12]: For reference: https://docs.ogc.org/is/17-069r3/17-069r3.html#_parameter_datetime @captaincoordinates

Comment (captaincoordinates) [reply order/README.md]: @m-mohr PR updated and includes some wording from the OGC Features API spec related to dates (which supports `..` or empty strings for open date ranges).

**Review by m-mohr — APPROVED** [review]

**Review by matthewhanson — APPROVED** [review]

**Pull request merged**